shared_model/interfaces/common_objects/string_view_types.hpp

```diff
@@ -9,12 +9,27 @@
 #include <string_view>
 
 #include <boost/serialization/strong_typedef.hpp>
+#include "interfaces/common_objects/byte_range.hpp"
 
 namespace shared_model {
   namespace interface {
     namespace types {
       BOOST_STRONG_TYPEDEF(std::string_view, SignedHexStringView)
+      BOOST_STRONG_TYPEDEF(ByteRange, SignatureByteRangeView)
+
       BOOST_STRONG_TYPEDEF(std::string_view, PublicKeyHexStringView)
+      BOOST_STRONG_TYPEDEF(ByteRange, PublicKeyByteRangeView)
+
+      template <
+          typename Dest,
+          typename Source,
+          typename Underlying = std::decay_t<decltype(std::declval<Dest>().t)>>
+      inline Dest makeStrongView(const Source &str) {
```

Comment:
Maybe place this function together with `ByteRange` instead, because it is unrelated to hex string view?